core/libdivecomputer.c

```diff
@@ -967,10 +967,20 @@ static void lookup_fingerprint(dc_device_t *device, device_data_t *devdata)
 {
 	char *cachename;
 	struct memblock mem;
+	unsigned char *raw_data;
```

Comment:
`const unsigned char *` - the fingerprint is not modified, is it?

Reply:
good catch